llvm/lib/Target/AMDGPU/SIRegisterInfo.td

```diff
@@ -900,6 +900,14 @@ def SReg_64 : SIRegisterClass<"AMDGPU", [v2i32, i64, v2f32, f64, i1, v4i16, v4f1
   let Size = 64;
 }
 
+def SReg_64_Plus : SIRegisterClass<"AMDGPU", [v2i32, i64, v2f32, f64, i1, v4i16, v4f16, v4bf16], 32,
```

Comment:
Class names need work. We probably should do a round of cleanup on all of these. How about SReg_64_Encodable?